internal/cmd/audit-log/command_config_update.go

```diff
@@ -16,7 +16,7 @@ import (
 func (c *configCommand) newUpdateCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "update",
-		Short: "Submits audit-log config spec object to the API.",
+		Short: "Submits audit-log configuration spec object to the API.",
```

Comment:
Same as @brianstrauch ^^ comment above about "spec" here.

Reply:
Hmm, I think we can keep it as specification for now.